diff --git a/Export.lean b/Export.lean
index 7a73832..66a3276 100644
--- a/Export.lean
+++ b/Export.lean
@@ -77,8 +77,8 @@ def removeMData (e : Expr) : M Expr := do
     pure <| e.updateApp! (← removeMData f) (← removeMData a)
   | .lam _ d b _ =>
     pure <| e.updateLambdaE! (← removeMData d) (← removeMData b)
-  | .letE _ d v b _ =>
-    pure <| e.updateLet! (← removeMData d) (← removeMData v) (← removeMData b)
+  | .letE _ d v b nondep =>
+    pure <| e.updateLet! (← removeMData d) (← removeMData v) (← removeMData b) nondep
   | .forallE _ d b _ =>
     pure <| e.updateForallE! (← removeMData d) (← removeMData b)
   | .proj _ _ e2 =>
